Rollbar/NSJSONSerialization+Rollbar.m

```diff
@@ -52,7 +55,7 @@ + (NSDictionary *)safeDataFromJSONObject:(id)obj {
             [safeData setObject:[[obj allObjects] componentsJoinedByString:@","] forKey:key];
         } else if ([obj isKindOfClass:[NSData class]]) {
             NSError* error = nil;
-            NSDictionary* json = [NSJSONSerialization JSONObjectWithData:obj options:kNilOptions error:&error];
+            NSMutableDictionary* json = [NSJSONSerialization JSONObjectWithData:obj options:kNilOptions error:&error];
```

Comment:
The result will actually be an `NSDictionary` not an `NSMutableDictionary` because of the `kNilOptions` being passed in. If you want a mutable dictionary you have to pass in different options. It doesn't really matter here but no reason to change this line.

Reply:
definitely missed the options. the intent was to do the same thing as in RollbarReadLogItemFromFile() implementation. i just had so many exceptions regarding attempts to mutate nonmutable objects during the development, so i wanted to make sure we only have mutable objects within payloads - always.

Reply:
for now, i added the intended options. or, you would still prefer to keep this line the way the very original one was?